src/components/pages/AdminDashboard/DevToolsNew.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import { Menu, Dropdown, Button } from 'antd';
+import { DownOutlined } from '@ant-design/icons';
+
+import { devMode } from '../../../state/actions/index';
+
+const DevToolsNew = ({ devMode, setDevMode }) => {
+  const { push } = useHistory();
+
+  const handleSatMon = e => {
```

Comment:
I think we can dynamically decide the day id here by adding a unique id or name to each menu item <a>, so we could pull that into this handler and not have to create several separate handlers.